From b6e7d6d901fa5c6fa54c0d9122176aa2bf735f2f Mon Sep 17 00:00:00 2001
From: Haohui Mai <wheat9@apache.org>
Date: Sun, 22 Nov 2015 22:58:58 -0800
Subject: [PATCH 1075/2848] HDFS-9356. Decommissioning node does not have Last
 Contact value in the UI. Contributed by Surendra
 Singh Lilhore.

(cherry picked from commit 5ce7e72145dd855b7c4e0d77ba80fe5c76397edc)

Change-Id: I99b1393d612dc6ff12fd68a5ec1a7c94174e963e
---
 .../src/main/webapps/hdfs/dfshealth.html           |    2 --
 1 file changed, 2 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
index 8937c00..d691a3e 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.html
@@ -340,7 +340,6 @@
   <thead>
     <tr>
       <th>Node</th>
-      <th>Last contact</th>
       <th>Under replicated blocks</th>
       <th>Blocks with no live replicas</th>
       <th>Under Replicated Blocks <br/>In files under construction</th>
@@ -349,7 +348,6 @@
   {#DecomNodes}
   <tr>
     <td>{name} ({xferaddr})</td>
-    <td>{lastContact}</td>
     <td>{underReplicatedBlocks}</td>
     <td>{decommissionOnlyReplicas}</td>
     <td>{underReplicateInOpenFiles}</td>
-- 
1.7.9.5

